config/prod/shinyserver-pro-prod-alb-v2.yaml

```diff
@@ -0,0 +1,22 @@
+# Provision a load balancer for shiny.synapse.org
+template_path: remote/managed-alb-https-v2.yaml
+stack_name: shinyserver-pro-prod-alb-v2
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastucture"
+  # The resource owner
+  OwnerEmail: "x.schildwachter@sagebase.org"
+
+  # load balancer configs
+  Ec2InstanceId: "i-0a41a3cbb9de74e9f"
```

Comment:
i would suggest you directly reference from the output of shinyserver-pro-prod stack like so..
```
Ec2InstanceId: !stack_output_external shinyserver-pro-prod::Ec2InstanceId
```

Reply:
Seen this, will do!